src/features/session_trace/aggregate/index.js

```diff
@@ -272,6 +274,8 @@ export class Aggregate extends AggregateBase {
   // Tracks the events and their listener's duration on objects wrapped by wrap-events.
   storeEvent (currentEvent, target, start, end) {
     if (this.shouldIgnoreEvent(currentEvent, target)) return
+    if (this.prevStoredEvents.has(currentEvent)) return // prevent multiple listeners of an event from creating duplicate trace nodes per occurrence
```

Comment:
Will the ST feature's emitters keep emitting if the feature becomes `blocked`? If so, this condition should also probably check if the feature is blocked before storing in the map to prevent data bloat

Reply:
Resolved in huddle